releasenotes/_deconst.json

```diff
@@ -1,5 +1,5 @@
 {
-    "contentIDBase": "https://github.com/rcbops/rpc-openstack/v13”,
+    "contentIDBase": "https://github.com/rcbops/rpc-openstack/master”,
```

Comment:
This file isn't valid JSON because it uses "smart quotes" instead of regular ones! Notice how GitHub's code formatting flags it in red.

For some reason Macs have smart quote replacement enabled by default; I highly recommend [turning it off.](http://www.iclarified.com/38772/how-to-disable-curly-quotes-in-mac-os-x-mavericks)